src/CounterInterceptor/MyActivities.cs

```diff
@@ -0,0 +1,19 @@
+namespace TemporalioSamples.CounterInterceptor;
+
+using System.Diagnostics;
+using Temporalio.Activities;
+
+public class MyActivities
+{
+    [Activity]
+    public string SayHello(string name, string title)
+    {
+        return "Hello " + title + " " + name;
+    }
```

Comment:
```suggestion
    public string SayHello(string name, string title) => $"Hello {title} {name}";
```

Use string interpolation and single-statement syntax (will stop commenting on this, but it applies project wide)

Reply:
Bump, this was not addressed